include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -326,4 +326,10 @@
 #    define _ONEDPL_STD_BIT_FLOOR_BROKEN 0
 #endif
 
+#if defined(_MSC_VER) && (_MSC_VER < 1939)
+#    define _ONEDPL_STD_RANGES_WINDOWS_NON_GLOBAL_FUN_OBJ 1
+#else
+#    define _ONEDPL_STD_RANGES_WINDOWS_NON_GLOBAL_FUN_OBJ 0
+#endif
```

Comment:
Could you comment the reason to add this macro? Its purpose is not obvious for a one who does not see the PR's description.

Also, this macro looks like a broken macro, because it is a workaround for an issue in the MSVC library. That is why I suggest renaming: `_ONEDPL_STD_RANGES_WINDOWS_NON_GLOBAL_FUN_OBJ` -> `_ONEDPL_STD_RANGES_FUN_OBJ_WINDOWS_BROKEN`.

Reply:
dont mind